november-2017/php-version.md

```diff
@@ -0,0 +1,6 @@
+### Upcoming MyBB Releases And Their Minimum Supported PHP Versions
+Frequent visitors to the MyBB Community Forums may remember [a discussion from July 2016](https://community.mybb.com/thread-195675.html) regarding the minimum PHP version required to use MyBB 2.0. The benefits of such a change were detailed in great length by [Devilshakerz](https://community.mybb.com/user-47371.html).
+
+With the recent announcement of the 1.10 series and the uncertainty surrounding *exactly* what is happening to MyBB 2.0, we believe that changes could and *should* happen as soon as 1.10. At the time of writing this blog post, the current stable release of MyBB is 1.8.13 and the minimum supported PHP version is 5.2. To put that into some perspective: PHP 5.2 was released on the 2nd of November 2006 and was then declared EOL (end-of-life) on the 6th January 2011. PHP 5.2 has served us well but has not received any performance or security updates in over 6 years, it actively hinders the development and modernisation of MyBB as a whole. Since then there have been several other PHP releases both made generally available and since reaching end of life status. The PHP team maintain [a list of versions that have reached End Of Life](http://php.net/eol.php) status that is kept up to date and provides exact dates for version lifetimes.
+
+In the interest of ensuring MyBB remains up to date with current best practices and to leverage the new features and optimisations generally found in new versions of PHP, we propose that the minimum PHP version requirement be increased from PHP 5.2 to either 5.6 or 7.0, although that exact version hasn't been decided yet. PHP 5.6 still has just over a year until it reaches EOL, becoming so on the 31st of December 2018. Another suggested pathway has been to leave the 1.8 series where it is, supporting PHP 5.2 indefinitely and instead pushing for PHP 7.0 as the requirement for the 1.10 series and even further aggresive version bumping at the eventual 2.0 development and release cycle. With 1.10 becoming a very real concept with each passing day (some PRs have entered the review stage and suggestions are being taken [here](https://community.mybb.com/forum-199.html)) we're really keen to gather as much feedback as possible and to learn from our userbase what their concerns might be when confronted with such a prodigious change.
```

Comment:
```diff
- although that exact version hasn't been decided yet
```

This statement isn't required and the sentence is long already. Removing it makes the paragraph easier to digest.